product-development/README.md

```diff
@@ -104,57 +117,99 @@ Example:
+### DRIs
 
-Dan and Nick will typically not be DRIs on epics, in order to encourage all Tilters to have more responsibility for innovation and execution of day to day work.
+Epics are assigned to a
+[DRI](https://medium.com/@mmamet/directly-responsible-individuals-f5009f465da4). The
+DRI is responsible for driving the epic to completion. An engineer should be the
+DRI of at most one `In Progress` epic at any time. An epic should only have one
+owner at any time.
 
-## Picking a high priority unstarted epic
-When picking a high priority unstarted epic, look at the epics _above the line_ in the [backlog](https://app.clubhouse.io/windmill/epics?state_ids=500008016&state_ids=500008001) and choose a sensible one based on at least these example scenarios:
-- My skill set is better suited for the third highest priority epic, so I'll pick that one, and save the first two for other folks who can finish them more quickly.
-- The fourth highest priority epic looks to be small so I'll pick that one first and finish it as a quick win before the end of the week.
-- My teammate has expertise in the highest priority epic and I'd like to collaborate and learn from them. But they aren't free just yet. I'll work on the third highest priority epic for now and plan to come back to pair with my teammate next week.
+- The DRI is responsible for maintaining the epic state, description, and attached stories updated as the single source of truth of overall work status, at least on a daily basis.
+- The acceptance criteria sections are _especially critical_, including presenting the problem and final implementation in the [weekly epics meeting](#weekly-epics-meeting).
+- The DRI is responsible for reducing scope if additional work is discovered.
+- The DRI is responsible for driving the epic to completion.
 
-Dan B and Nick, and Victor are responsible for setting the location of the line.
+### How DRIs are Assigned
 
-Victor is responsible for tracking [open customer requests](https://companies-b164c.firebaseapp.com/customer-requests) as an input for prioritization.
+When assigning DRIs, we're operating on a few hypotheses:
+
+- Projects are far more likely to succeed when the person working on it feels
+  connected to the project, and feels it lets them use their skills well.
+
+- Conversely, Tilters should be empowered to try epics they feel strongly about,
+  even if other people on the team disagree. Consensus is a bad way to make
+  decisions, and we shouldn't require consensus on an idea to move forward.
+  
+- It's anxiety-producing and not helpful when people are constantly switching
+  back and forth between execution and prioritization. We don't want each DRI
+  to have to do a prioritization exercise every week when they finish their epic.
+  
+So we're trying a new system.
+
+Each engineer should have a personal backlog of epics that they want to work on.
+
+It doesn't matter what form this backlog takes. It can be a shared Google Doc, or a
```

Comment:
@nicks : What is this trying to address? Transparency, more freedom?

- If everyone has their own format of their backlog of work, I'm worried that it would be difficult to organize ideas, move them around to different DRIs (as you mentioned earlier), since essentially the ideas are not compatible in format.
- We already have a lot of floating Google docs around for various initiatives and ideas in various stages of ideation/development/execution. I'm not fundamentally opposed to this, but it does make it harder to see at a glance, all the ongoing things at Tilt. If everyone has their own backlog of work in their own format, I'm worried it would add more overhead.
- If we are using pre-assigning epics to individuals, then we can just filter on the Owner field in Clubhouse to dynamically see people's backlogs. 
- The benefit of having a personal backlog is that you could even create placeholder epics or account management epics in your own backlog. So everyone can at a glance see your work. This would be a bit harder to navigate if every person maintains their own google doc or spreadsheet.

Reply:
ya, this is a good criticism, let me change this section to express this a little bit better.

i don't think we want each person on the team to have a personal, curated backlog

i also want to avoid the situation where a person does zero prep work (e.g., reading the existing backlog, thinking about what connects to their values/skills, articulating what they want), and expects that they will get assigned something fulfilling that they can do well. I've worked with engineers like this (google has a bunch) and i don't think it's a healthy expectation.

but i do think we can strike a balance where people affirm what they want to work on without making that the only input into the decision.

Reply:
@victorwuky ok, rewrote this a bit, i think calling this a "backlog" was confusing, what do you think of it now?

Reply:
Latest looks great. Thanks @nicks . I approved.